web/src/message_list_view.js

```diff
@@ -183,6 +183,21 @@ function set_topic_edit_properties(group, message) {
     }
 }
 
+function get_users_for_recipient_row(message) {
+    const user_ids = people.pm_with_user_ids(message);
+    return user_ids
+        .map((user_id) => {
+            const person = people.get_by_user_id(user_id);
+            if (people.should_add_guest_user_indicator(user_id)) {
+                return $t_html({defaultMessage: "{name} <i>(guest)</i>"}, {name: person.full_name});
+            }
+            // Escaping here is necessary because we don't do escaping in Handlebar template
+            // for this.
+            return _.escape(person.full_name);
+        })
+        .join(", ");
+}
```

Comment:
I really don't like using `{{{` as every time we use it adds security risk if it ends up being used incorrectly. Is there a reason we can't just pass objects into the recipient row rendering and have the loop be in Handlebars?

Reply:
Are you asking how to do `{{#unless @last}},{{/unless}}` (documentation: [`@last`](https://handlebarsjs.com/api-reference/data-variables.html#last))?

Reply:
No. I am basically not sure about how to handle translation here. I guess we cannot just tag "You and" for translation and then add comma separated names. And I think we also cannot add handlebars inside `{[#tr}}`.